lib/mix/test/fixtures/configs/nested_import.exs

```diff
@@ -0,0 +1,3 @@
+use Mix.Config
+Mix.Config.eval!(Path.join([__DIR__, "good_config.exs"]))
+:done
```

Comment:
Do we use `:done` for anything here?

Reply:
I copied it from https://github.com/Gazler/elixir/blob/a7787e6077e2d5df7625509fd3c776b65d6bc9ec/lib/mix/test/fixtures/configs/good_import.exs

Reply:
Yeah I was wondering if we use that one for anything also 😄 Let's merge this and we'll check later. 👍